Dockerfile.core

```
@@ -0,0 +1,46 @@
+ARG OS_TAG=20.04
+FROM ubuntu:${OS_TAG} as builder
+
+ARG OS_TAG
+ARG BUILD_TYPE=release
+ARG DEBIAN_FRONTEND=noninteractive
```

Is it an `ARG` on purpose?

```suggestion
ENV DEBIAN_FRONTEND=noninteractive
```

(Or just add it in the relevant `RUN` commands like we do in Inspektor Gadget.)

---

It's a copy-pasta error. Anyway I'll remove this file.